config/config.go

```diff
@@ -493,6 +493,7 @@ type Performance struct {
 	PlanReplayerGCLease  string `toml:"plan-replayer-gc-lease" json:"plan-replayer-gc-lease"`
 	GOGC                 int    `toml:"gogc" json:"gogc"`
 	EnforceMPP           bool   `toml:"enforce-mpp" json:"enforce-mpp"`
+	TiFlashMaxThreads    int `toml:"tiflash-max-threads" json:"tiflash-max-threads"`
```

Comment:
I think that this should also just be converted to a ScopeGlobal variable, and not added to the configuration file. Previously, we didn't have clear criteria for what should be in the config file vs. a system variable, but it has become more clear the last few months. The PM (Eason) has asked that we always add `GLOBAL` variables instead, because it simplifies cluster configuration.

See https://github.com/pingcap/tidb/issues/33769 for examples where we are having to convert configuration options to system variables. The problem with using the config file is the cloud team needs to update the config file on each node and restart the tidb-server (complex).

Reply:
> I think that this should also just be converted to a ScopeGlobal variable, and not added to the configuration file. Previously, we didn't have clear criteria for what should be in the config file vs. a system variable, but it has become more clear the last few months. The PM (Eason) has asked that we always add `GLOBAL` variables instead, because it simplifies cluster configuration.
> 
> See #33769 for examples where we are having to convert configuration options to system variables. The problem with using the config file is the cloud team needs to update the config file on each node and restart the tidb-server (complex).

Ok, it will be only converted to a ScopeGlobal variable.